styles/styles.css

```diff
@@ -303,9 +303,17 @@ main > .section > .default-content-wrapper > p {
   margin: auto;
   max-width: 60%;
   text-align: left;
+  margin-top: 5vh;
 }
 
-.blog main > .section > .default-content-wrapper > p {
+.blog main > .section > .default-content-wrapper > p:first-of-type {
+  font-family: 'GFS Didot', serif;
```

Comment:
Why only the first-of-type? What if Mike authors more than one paragraph?

Reply:
Yes that's why I was recommending "updated" information to be in metadata table rather. That way its styling would have been fixed automatically along with Autoblocked items. @helms-charity

Reply:
@helms-charity  - When we updated your blog sheet with the updated information, it was coming as a P element - hence we need to update that P - first-of-type . If that helps further.

Reply:
Remove this section about 14px please, we don't need it. I will address it another way.

Reply:
@helms-charity  - NM, I figured out a way via CSS itself.